### PR TITLE
exfatlabel: add page

### PR DESCRIPTION
**Pull request by Managor**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

File: pages/common/exfatlabel.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# exfatlabel
+
+> Set an exfat filesystem label.
+> More information: <https://www.mankier.com/8/exfatlabel>.
```
Comment (kbdharun) [inline pages/common/exfatlabel.md:4]:
```suggestion
> More information: <https://manned.org/exfatlabel>.
```
We prefer using manned when there isn't any documentation from the author. (Ref. https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#more-information-links)

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have a minor suggestion for this page.

File: pages/common/exfatlabel.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# exfatlabel
+
+> Set an exfat filesystem label.
```
Comment (kbdharun) [inline pages/common/exfatlabel.md:3]:
```suggestion
> Set an exFAT filesystem label.
```

File: pages/common/exfatlabel.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+> More information: <https://www.mankier.com/8/exfatlabel>.
+
+- Set a filesystem label:
+
+`exfatlabel /dev/{{sda}} "{{new label}}"`
```
Comment (kbdharun) [inline pages/common/exfatlabel.md:8]:
```suggestion
`exfatlabel /dev/{{sda}} "{{new_label}}"`
```
To be consistent with you `mlabel` PR.

**Review by kbdharun — APPROVED** [review]

File: pages/common/exfatlabel.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
+# exfatlabel
+
+> Set an exFAT filesystem label.
+> More information: <https://manned.org/exfatlabel>.
+
+- Set a filesystem label:
+
+`exfatlabel /dev/{{sda}} "{{new_label}}"`
```
Comment (acuteenvy) [inline pages/common/exfatlabel.md:8]:
```suggestion
- Display the current filesystem label:

`exfatlabel {{/dev/sda}}`

- Set the filesystem label:

`exfatlabel {{/dev/sda}} {{new_label}}`
```

**Review by acuteenvy — APPROVED** [review]

**Pull request merged**